### ISSUE TITLE
Clean up non-functional components

### ISSUE THREAD

**Issue by annaritz** [Reed-CompBio/spras#29]
Once everything is merged into master from Pull Request #27 , I would like to remove the non-functional components of code.  This includes:
- Remove `augmentation_settings` config labels, code in `parse_config()` function, and code in `Snakemake` file.
- Remove `ensemble` code in `parse_config()` function and code in `Snakemake` file.
- Remove algorithms that are not yet implemented. (Keep OmicsIntegrator1 & 2 and PathLinker).
- Remove `evaluation_settings` config labels (which set up the precision-recall code). This should go under `analysis`. We will need to carefully think about the options we need here.
- Remove `plot_settings` config labels (which sets up the precision-recall plotting). This should go under `analysis`. We will need to carefully think about the options we need here.

Pull Request #27 includes a `config/config_prototype.yaml` file that contains an older and complete config file, so we have a record of how we envisioned all these components working together.
